source/ingress.go

```diff
@@ -145,26 +146,18 @@ func (sc *ingressSource) endpointsFromTemplate(ing *v1beta1.Ingress) ([]*endpoin
 
 	hostname := buf.String()
 
-	endpoints = getEndpointsFromTargetAnnotation(ing, hostname)
-
-	if len(endpoints) != 0 {
-		return endpoints, nil
-	}
-
 	ttl, err := getTTLFromAnnotations(ing.Annotations)
 	if err != nil {
 		log.Warn(err)
 	}
-	for _, lb := range ing.Status.LoadBalancer.Ingress {
-		if lb.IP != "" {
-			endpoints = append(endpoints, endpoint.NewEndpointWithTTL(hostname, lb.IP, endpoint.RecordTypeA, ttl))
-		}
-		if lb.Hostname != "" {
-			endpoints = append(endpoints, endpoint.NewEndpointWithTTL(hostname, lb.Hostname, endpoint.RecordTypeCNAME, ttl))
-		}
+
+	targets := getTargetsFromTargetAnnotation(ing)
```

Comment:
here you use the function as far as I see, so drop the fixme :)

Reply:
Still, I removed the unnecessary comment :)